apps/src/dance/types.ts

```diff
@@ -41,3 +41,20 @@ export enum AiOutput {
   GENERATED_BLOCKS = 'generated_blocks',
   BOTH = 'both',
 }
+
+// The first item is user readable / translated string (eg, "Blooming Petals"),
+// and the second item is english-only key (eg, "blooming_petals").
+// Example: ['Blooming Petals', 'blooming_petals']
+export type TranslationTuple = [string, string];
```

Comment:
Could we just use Blockly's `MenuOption` type here? That way we don't have to cast the result of `dropdown.getOptions()`?

Reply:
Well, I think it's less of a typescript thing and more just providing clarity around what the data is representing? In this case, it's really just the list of dropdown options we're after, right? I wonder if actually just calling these "dropdown labels" rather than "translations" might be clearer in terms of what we're actually reading - but that might just be me

Reply:
Agreed re: variable name being confusing, updated here! https://github.com/code-dot-org/code-dot-org/pull/54534/commits/b3af2873f940b7308f8d764ce13ab126130a36e6